t/10-jobs.t

```diff
@@ -633,13 +633,7 @@ subtest 'job with skipped modules' => sub {
 sub job_is_linked {
     my ($job) = @_;
     $job->discard_changes;
-    my $comments = $job->comments;
```

Comment:
It would be better not to check every comment for a label and having to create DBIx objects for every comment like the current code does.

Reply:
Since it's in a test and we don't have thousands of jobs, it's probably ok

Reply:
Martchus is still right and code in tests can act as examples for someone to use in the future. We can use the same approach as in 3ccb16bdb though